openassessment/templates/openassessmentblock/oa_uploaded_file.html

```diff
@@ -20,7 +20,7 @@ <h3 class="{{ class_prefix }}__display__title">
                 {% trans "View the file associated with this submission." %}
             </a>
             {% if show_warning %}
-                <p class="submission_file_warning">{% trans "(Caution: This file was uploaded by another course learner and has not been verified, screened, approved, reviewed, or endorsed by edX. If you decide to access it, you do so at your own risk.)" %}</p>
+                <p class="submission_file_warning">{% trans "(Caution: This file was uploaded by another course learner and has not been verified, screened, approved, reviewed, or endorsed by the platform maintainer. If you decide to access it, you do so at your own risk.)" %}</p>
```

Comment:
I don't think typical users will know what a "platform" is.  "site administrator" seems like something people will recognize.